rasterio/features.py

```diff
@@ -125,8 +125,7 @@ def shapes(source, mask=None, connectivity=4, transform=IDENTITY):
         source = source.data
 
     transform = guard_transform(transform)
-    for s, v in _shapes(source, mask, connectivity, transform):
-        yield s, v
+    yield from _shapes(source, mask, connectivity, transform)
```

Comment:
We're planning to run pyupgrade on the codebase (#3058), so I've tried it out on the files changed in this PR.